crates/agglayer-jsonrpc-api/src/testutils.rs

```diff
@@ -218,34 +333,18 @@ impl Drop for TestContext {
 }
 
 #[fixture]
-pub(crate) async fn context() -> TestContext {
+pub async fn context() -> TestContext {
     TestContext::new().await
 }
 
 #[fixture]
-pub(crate) async fn raw_rpc() -> RawRpcContext {
+pub async fn raw_rpc() -> RawRpcContext {
     TestContext::new_raw_rpc().await
 }
 
-pub(crate) fn next_available_addr() -> std::net::SocketAddr {
-    use std::net::{TcpListener, TcpStream};
-
-    assert!(
-        std::env::var("NEXTEST").is_ok(),
-        "Due to concurrency issues, the rpc tests have to be run under `cargo nextest`",
-    );
-
-    let host = "127.0.0.1";
-    // Request a random available port from the OS
-    let listener = TcpListener::bind((host, 0)).expect("Can't bind to an available port");
-    let addr = listener.local_addr().expect("Can't find an available port");
-
-    // Create and accept a connection (which we'll promptly drop) in order to force
-    // the port into the TIME_WAIT state, ensuring that the port will be
-    // reserved from some limited amount of time (roughly 60s on some Linux
-    // systems)
-    let _sender = TcpStream::connect(addr).expect("Can't connect to an available port");
-    let _incoming = listener.accept().expect("Can't accept an available port");
-
-    addr
+pub fn next_available_addr() -> std::net::SocketAddr {
+    std::net::TcpListener::bind("127.0.0.1:0")
+        .unwrap()
+        .local_addr()
+        .unwrap()
```

Comment:
I just checked and this does not seem to be reliable with `cargo test`, emitting an "Address already in use" error. Would prefer the check for nextest to stay in place.